Bang & Olufsen add overlay/announce play_media documentation (#31879)
home-assistant/home-assistant.github.io: source/_integrations/bang_olufsen.markdown

### play_media services

> The Bang & Olufsen integration supports different playback types in the `media_player.play_media` service: playback from URL, activating a favourite, playback from a local file, playing a radio station, activating a Deezer flow and Deezer playlists, albums, and tracks and playing files and TTS as an overlay.

Reviewer: ```suggestion
The Bang & Olufsen integration supports different playback types in the `media_player.play_media` service: playback from URL, activating a favorite, playback from a local file, playing a radio station, activating a Deezer flow and Deezer playlists, albums, tracks, and playing files and text-to-speech (TTS) as an overlay.
```

  media_content_id: 1234567890
```

##### Overlay

> Interrupt currently playing media to play an audio message.

Reviewer: ```suggestion
Interrupts currently playing media to play an audio message.
```

> To use the Bang & Olufsen Cloud TTS, use `overlay_tts` as the `media_content_type` and enter message in the `media_content_id` field.

Reviewer: ```suggestion
To use the Bang & Olufsen Cloud TTS, use `overlay_tts` as the `media_content_type` and enter a message into the `media_content_id` field.
```

Bang & Olufsen Cloud TTS messages are limited to 100 unique messages a day and are cached for 24 hours.

Extra keys available:

| Service data attribute    | Optional | Description                                                      |
| ------------------------- | -------- | ---------------------------------------------------------------- |
| `overlay_absolute_volume` | yes      | Specify an absolute volume for the overlay.                      |
| `overlay_offset_volume`   | yes      | Specify a volume offset to be added to the current volume level. |

> | `overlay_tts_language`    | yes      | Specify TTS language.                                            |

Reviewer: ```suggestion
| `overlay_tts_language`    | yes      | Specify the language used for text-to-speech.                                            |
```

> | `overlay_tts_language`    | yes      | Specify TTS language.                                            |

Reviewer: Would it make sense to mention what type of code is used to specify the language? Is there a default, en-us?

Author: I've added the default language and the language standard.

> | `overlay_tts_language`    | yes      | Specify the language used for text-to-speech.                                            |

Reviewer: Would it make sense to mention what type of code is used here to specify the language? Is there a default, en-us?

Extra keys available:

| Service data attribute    | Optional | Description                                                                                      |
| ------------------------- | -------- | ------------------------------------------------------------------------------------------------ |
| `overlay_absolute_volume` | yes      | Specify an absolute volume for the overlay.                                                      |
| `overlay_offset_volume`   | yes      | Specify a volume offset to be added to the current volume level.                                 |

> | `overlay_tts_language`    | yes      | Specify the language used for text-to-speech. Uses the BCP 47 standard. Default value is "en-us" |

Reviewer: ```suggestion
| `overlay_tts_language`    | yes      | Specify the language used for text-to-speech. Uses the BCP 47 standard. Default value is "en-us". |
```
